<launch>
    <rosparam file="$(find cam2lidar)/config/temporal.yaml"/>
    <node pkg="cam2lidar" name="temporal_calibration" type="temporal_calibration.py" output="screen">
        <param name="image_topic" type="str" value="/camera/color/image_raw"/>
        <param name="camera_info_topic" value="/camera/color/camera_info"/>
        <param name="lidar_topic" type="str" value="/velodyne_points"/>
        <!-- <param name="distance_threshold" value="15"/> -->
        <!-- <param name="consequent_frames" value="15"/> -->
        <param name="debug" value="False"/>
    </node>
    <node pkg="cam2lidar" name="Calibration_visualization" type="user_interface.py" output="screen">
        <param name="subscriber_name" value="/temporal_visualization"/>
        <param name="camera_info_topic" value="/camera/color/camera_info"/>
    </node>
</launch>